src/Telegram/README.md

```diff
@@ -8,13 +8,20 @@ composer require socialiteproviders/telegram
 
 Please see the [Base Installation Guide](https://socialiteproviders.com/usage/), then follow the provider specific instructions below.
 
-### Add configuration to `config/services.php`
+## Configuration
+
+First of all, you must create a bot by contacting [@BotFather](http://t.me/BotFather) (https://core.telegram.org/bots#6-botfather)
+
+> Don't forget to set your website URL using `/setdomain`
+
+Then, you need to add your bot's configuration to `config/services.php`. The bot username is required, `client_id` must be `null`. The provider will also ask permission for the bot to write to the user.
 
 ```php
-'telegram' => [    
-  'bot' => env('TELEGRAM_BOT'),  
-  'client_secret' => env('TELEGRAM_CLIENT_SECRET'),  
-  'redirect' => env('TELEGRAM_REDIRECT_URI') 
+'telegram' => [
+    'bot' => env('TELEGRAM_BOT_NAME'),  // The bot's username
+    'client_id' => null,
```

Comment:
guess we can remove this config option?

Reply:
I think removing `client_id` would raise an exception because of https://github.com/SocialiteProviders/Manager/blob/master/src/ConfigTrait.php#L23

Reply:
Ahh my bad, ignore :P

Reply:
No problem 😁